plenum/server/consensus/replica_service.py

```diff
@@ -42,30 +44,47 @@ def __init__(self, name: str, validators: List[str], primary_name: str,
         self._network = network
         self._data = ConsensusSharedData(name, validators, 0)
         self._data.primary_name = generateName(primary_name, self._data.inst_id)
+        self._timer = timer
         self.config = getConfig()
         self.stasher = StashingRouter(self.config.REPLICA_STASH_LIMIT, buses=[bus, network])
         self._write_manager = write_manager
         self._primaries_selector = RoundRobinNodeRegPrimariesSelector(self._write_manager.node_reg_handler)
+
+        self._freshness_checker = FreshnessChecker(freshness_timeout=self.config.STATE_FRESHNESS_UPDATE_INTERVAL)
+        for ledger_id in [POOL_LEDGER_ID, DOMAIN_LEDGER_ID, CONFIG_LEDGER_ID]:
```

Comment:
This needs to be done for Plugin ledgers as well. 
Also please remove similar code from the `replica.py` if needed.

Reply:
This is a ReplicaService, which is used in simulation tests only, so as long as we don't test plugin ledgers it is enough to use just this hard-coded array (and if we do - then much more work would be needed, for example either porting node itself to be usable in simulation tests or making plugin register in a place accessible to simulation tests). Also, for the same reason we shouldn't remove freshness checker from replica code, which is used in production